src/components/Scroll.js

```diff
@@ -21,18 +38,31 @@ class Scroll extends Component {
 
   static propTypes = {
     children: PropTypes.node.isRequired,
+    arrayLength: PropTypes.number.isRequired,
   }
 
   render() {
-    return (
-      <div className="OuterDiv">
-        <button className="leftbutton scroll-button" onClick={this.scrollLeft}><FontAwesomeIcon icon="angle-left" /></button>
-        <div className="scrolling-wrapper-flexbox" ref={this.inputRef}>
-          {this.props.children}
+    let result = <div className="OuterDiv" />;
+    if ((this.props.arrayLength * 170) < (this.state.width - 250)) {
```

Comment:
What is `170` and `250`? Assign these to some variables to make it clearer what they represent.